code/datums/components/space_kidnap.dm

```diff
@@ -63,3 +63,11 @@
 
 		var/obj/wisp = new /obj/effect/wisp_mobile (get_turf(pick(GLOB.voidwalker_void)))
 		victim.forceMove(wisp)
+		succesfully_kidnapped()
+
+/datum/component/space_kidnap/proc/succesfully_kidnapped()
+	var/obj/item/void_eater/eater = locate(/obj/item/void_eater) in parent
+	eater?.refresh()
+
+	var/atom/owner = parent
+	owner.balloon_alert(owner, "void eater refreshed!")
```

Comment:
`!` is reserved for failure messages
may be better as a to_chat and a sound in any case, and/or a brief animated outline shader on the void eater inhand item as an attract animation